FirebaseRemoteConfig/Sources/RCNPersonalization.h

```diff
@@ -19,16 +19,27 @@
 NS_ASSUME_NONNULL_BEGIN
 
 static NSString *const kAnalyticsOriginPersonalization = @"fp";
-static NSString *const kAnalyticsPullEvent = @"_fpc";
-static NSString *const kArmKey = @"_fpid";
-static NSString *const kArmValue = @"_fpct";
+
+static NSString *const kAnalyticsPullEvent = @"personalization_assignment";
```

Comment:
Minor: as I commented on the Android change, it's unclear to me what "pull" means here and below.

Also, given this whole file concerns logging to Analytics, including "Analytics" in some field names, but not others is confusing. I'd vote to just omit it.

The main distinctions from my perspective are internal vs external, and "event" vs "param" (to use Analytics' "feature type" nomenclature from the [event proposal](http://go/p13n-events)).

To summarize, I'd vote to use "internal" and "external" prefixes to differentiate internal from external events, and "event" and "param" suffixes to differentiate event names from param names, eg `kInternalEvent`, `kInternalChoiceIdParam`, etc

Thinking (no action req'd): all these field names are consistent across Android and iOS 👍